### ISSUE TITLE
Verifying in Verify by file mode

### ISSUE THREAD

**Issue by dreams-togo** [FreeUKGen/ComETT#92]
The Identity settings were set as Verify by File and I did my first scan transcribe as such. However,  I didn&#039;t see any option to Validate. When I try to upload the file it states that it&#039;s not verified.
When I switch to Verify line-by-line then I get the verify button after each submit. So far therefore I&#039;m unable to upload my first transcription file

**Comment by DeniseColbert** [comment]
There appears to be a bug that is preventing upload of a file when user is in 'Verify by File' mode.

**Comment by stsccfr** [comment]
The Epic for all issues related to Verify is #131.  Other issues are #114, #109 and #92.  In #114 the request was made to drop Verify altogether.  This would be good to do for the next round of testing, because FreeREG transcribers are not accustomed to the extra step imposed by having to verify each line even after it has been entered.  I would assign this Critical also.

**Comment by nurifreembd** [comment]
@stsccfr thank you for the details. Can this epic be linked to our epic transcriber to create a manual assignment for tracking purposes, end to end testing and release management please?

**Comment by stsccfr** [comment]
No, this is Verify issue is not related to assignment creation.  This
Verification has to do with checking a record that has just been
transcribed and Submitted, and affirming that it has been transcribed
correctly.

**Comment by nurifreembd** [comment]
@stsccfr Dale, thanks very much for the clarification.

**Comment by stsccfr** [comment]
Now that Verify has been dropped altogether, this issue can be closed.

**Comment by nurifreembd** [comment]
Done and thanks
